### PR TITLE
Use automated integration tests

### PR DESCRIPTION
**Pull request by kbalk**
Uses tardigrade-ci's ability to run automated integration tests against a mock AWS stack.

### CODE REVIEW

File: py_getter/__init__.py [hunk 1]
```diff
@@ -120,7 +116,8 @@ def cli():
     parser.add_argument("URI")
     parser.add_argument("PATH")
     parser.add_argument("--refresh", action="store_true")
+    parser.add_argument("--s3_endpoint_url")
```
Comment (lorengordon) [inline py_getter/__init__.py:119]: Just a nit to use a hyphen here as separator for cli args...

```suggestion
    parser.add_argument("--s3-endpoint-url")
```

**Review by lorengordon — COMMENTED** [review]
Just one nit, looking good overall. And I do think we need a changelog entry and minor version bump, as we are adding the `s3_endpoint_url` argument as a feature for the module.

**Comment by kbalk** [comment]
A changelog doesn't exist.  I could create one starting with this release ... I don't think I could go back and recreate that history with any level of confidence.

**Comment by lorengordon** [comment]
Oh good point, that's fine. Just a minor version bump then!

**Review by lorengordon — APPROVED** [review]
🚢 !

**Pull request merged**